build.yaml

```diff
@@ -3,4 +3,4 @@ targets:
     builders:
       dart_mappable_builder:
         options:
-          generateMethods: [decode, encode, stringify]
+          generateMethods: [decode, encode, stringify, copy]
```

Comment:
Why do we need copy methods?

Reply:
Same as exposing mappers, if we can afford this at no cost, it shouldn't be a question to generate them.